benchmarking/smartseq2/merge_picard_metrics/merge_picard_mets.py

```diff
@@ -0,0 +1,106 @@
+from crimson import picard
+import pandas as pd
+import numpy as np
+from google.cloud import storage
+import json
+from os.path import basename
+import sys
+import requests
+import argparse
+
+
+def retrieve_workflow_outputs(uuid, run_name):
+    # load cromwell credential
+    logins = json.load(open('/usr/secrets/broad-dsde-mint-dev-cromwell.json'))
+    # meta_url
+    metadata_url = "https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1/" + uuid + "/metadata?expandSubWorkflows=false"
+    r = requests.get(
+        metadata_url,
+        auth=(logins['cromwell_username'], logins['cromwell_password']))
+    data = r.json()
+    # load output files
+    files = data['outputs'][run_name]
+    return (files)
+
+
+def merge_picard_metrics(files, metric_name):
+    """
+    piepline output picard QC metrics at sinle cell/sample level.
+    This functin is called to merge/aggregate QC metrics by metrics type and then merge multiple QC measurement 
+    into single matrix file. In this file, column is sample/cell and row is QC metrics
+    param files: metric files from pipeline outputs
+    param met_name: metrics name with workflow name and subworkflow name as prefix. such as 'run_pipelines.RunStarPipeline.alignment_summary_metrics'
+    """
+    # set up auth
+    client = storage.Client()
```

Comment:
This could reuse the function I described above to download Cromwell outputs from `gs` here.